resources/assets/scripts/routes/archive.js

```diff
@@ -62,6 +67,63 @@ export default {
         document.forms.filters.submit();
       };
     });
+
+    // Save search
+    const saveSearchButton = document.getElementById('save-search');
+
+    if (saveSearchButton) {
+      let savedSearches = localStorage.getItem('saved-searches');
+      savedSearches = savedSearches ? JSON.parse(savedSearches) : {};
+
+      if (Object.entries(savedSearches).length === 50) {
+        saveSearchButton.onclick = () => {
+          addNotification(
+            __('Maximum number of saved searches reached', 'coop-library'),
+            sprintf(__('You have reached the maximum amount of saved searches (50). To save more, you must <a href="%s">delete some saved searches</a>.', 'coop-library'), CoopLibrary.savedSearchesLink),
+            'error'
+          );
+        };
+      } else {
+        new Pinecone.Dialog( saveSearchButton, {
+          title: __('Save search', 'coop-library'),
+          question: __('To save your search, please give it a name so you can identify it later.', 'coop-library'),
+          input: 'name',
+          inputLabel: __('Name of saved search:', 'coop-library'),
+          confirm: __('Save', 'coop-library'),
+          dismiss: __('Don&rsquo;t save', 'coop-library'),
+          callback: function callback(input) {
+            try {
+              const tags = document.querySelectorAll('.filters input:checked');
+              let savedSearches = localStorage.getItem('saved-searches');
+              savedSearches = savedSearches ? JSON.parse(savedSearches) : {};
+              const now = Date.now();
+              const term = document.querySelector('.filters input[name="s"]').value;
+              const name = input ? input : sprintf(__('Saved search for “%s”', 'coop-library'), term);
+              const url = window.location.href;
+              const filters = [];
+              Array.prototype.forEach.call(tags, tag => {
+                const label = document.querySelector(`[for="${tag.id}"]`);
+                filters.push(label.innerText);
+              });
+              savedSearches[now] = {name, term, url, filters};
+              localStorage.setItem('saved-searches', JSON.stringify(savedSearches));
+              const remaining = 50 - Object.keys(savedSearches).length;
+              addNotification(
+                __('Search saved', 'coop-library'),
+                sprintf(
+                  __('You have successfully saved this search. You can save %d more. You can see this search in your <a href="%s">saved searches page</a>.', 'coop-library'),
```

Comment:
The link to the Saved Searches page does not seem to be correct. In my case, the link goes to the first resource in the saved search instead of the Saved Searches page.

Reply:
The link expects the save searches page to be at `/my-resources/saved-searches`. Can you ensure that your page is at that location and try again?

Reply:
That worked. Thanks!